Doc/library/__main__.rst

```diff
@@ -1,25 +1,212 @@
-
-:mod:`__main__` --- Top-level script environment
-================================================
+:mod:`__main__` --- CLIs, import-time behavior, and ``__name__ == '__main__'``
```

Comment:
I prefer the previous title, because it sums up the topic in one short sentence.

Reply:
The former title is jargon, and I don't feel that it's communicating much on it's own. What is a top-level script? Or, is it talking about a script environment? Is there a different environment compared to a script environment? What might that be? Are there different levels of script environments?

I might be missing something, and maybe this term is just not known to *me*. Can you link to a general article about this term in question, "top-level script environments"? In my experience, it's not a familiar concept. By contrast the three clauses in the new title are both familiar and relevant.

What do you think? Do you maybe see a middle-ground?